eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -209,6 +209,7 @@ begin {
         "azure-internet-analyzer",
         "azure-iot",
         "azure-iot-central",
+        "azure-iot-device-update",
```

Comment:
I don't see this doc slug at https://review.docs.microsoft.com/en-us/help/contribute/metadata-taxonomies?branch=main#product is this the correct value?